buildSrc/src/main/kotlin/ClientConfig.kt

```diff
@@ -7,7 +7,8 @@ enum class ConfigFields(val defaultValue: String) {
     API_BASE_URL(""""https://staging-nginz-https.zinfra.io""""),
     ACCOUNTS_URL(""""https://wire-account-staging.zinfra.io""""),
     WEB_SOCKET_URL(""""https://staging-nginz-ssl.zinfra.io/await?client=""""),
-    SUPPORT_URL(""""https://support.wire.com"""")
+    SUPPORT_URL(""""https://support.wire.com""""),
+    SENDER_ID(""""782078216207"""")
```

Comment:
Should it be a constant value? 🤔

Reply:
we have 2 different sender IDs for the staging and prod , but I wanted to make sure it's working first and now I will update the staging id after creating the app in the console

Reply:
there will be different one for staging later , but as of now we don't have different configs for different builds , so it's just one , so that's why it's here and not a constant